### ISSUE TITLE
User specified not to request a Baseband ticket.

### ISSUE THREAD

**Issue by AntikerTa** [tihmstar/futurerestore#130]
Hello. With my iPhone 6 I always get this error, if I want to back up the SHSH Blobs with tsschecker.

I do not use a virtual machine. I use Mac OS on my iMac. From my iPad Air 2 and my 3 iPhone 7, I can back up the SHSH Blobs without error.

But with my iPhone 6 it does not work without these messages. Is this a Bug in Tsschecker? Thank you. 

Mein-iMac:10.3.1 10.3.2SHSH TSSchecker x$ ./tsschecker_macos -d iPhone7,2 -e MYECID -i 10.3.2 --buildid 14F89 -s
Version: 7ce951430c1e3a24a4918d77a443d2d8cd074cf3 - 234
[TSSC] manually specified ecid to use, parsed "MYECID" to dec:MYECID hex:MYECID
[TSSC] opening firmware.json
[JSON] counting elements
[JSON] parsing elements
[TSSC] got firmwareurl for iOS 10.3.2 build 14F89
[TSSC] opening Buildmanifest for iPhone7,2_10.3.2_14F89
[TSSR] Request URL set to https://gs.apple.com/TSS/controller?action=2
[TSSR] Sending TSS request attempt 1... success
also requesting APTicket for installType=Update
[TSSR] Request URL set to https://gs.apple.com/TSS/controller?action=2
[TSSR] Sending TSS request attempt 1... success
[TSSR] User specified not to request a Baseband ticket.
[TSSR] Request URL set to https://gs.apple.com/TSS/controller?action=2
[TSSR] Sending TSS request attempt 1... failure
[TSSR] Request URL set to https://17.111.103.65/TSS/controller?action=2
[TSSR] Sending TSS request attempt 2... failure
[TSSR] Request URL set to https://17.111.103.15/TSS/controller?action=2
[TSSR] Sending TSS request attempt 3... failure
[TSSR] Request URL set to http://gs.apple.com/TSS/controller?action=2
[TSSR] Sending TSS request attempt 4... failure
[TSSR] Request URL set to http://17.111.103.65/TSS/controller?action=2
[TSSR] Sending TSS request attempt 5... failure
[TSSR] Request URL set to http://17.111.103.15/TSS/controller?action=2
[TSSR] Sending TSS request attempt 6... failure
[TSSR] Request URL set to https://gs.apple.com/TSS/controller?action=2
[TSSR] Sending TSS request attempt 7... failure
[TSSR] Request URL set to https://17.111.103.65/TSS/controller?action=2
[TSSR] Sending TSS request attempt 8... failure
[TSSR] Request URL set to https://17.111.103.15/TSS/controller?action=2
[TSSR] Sending TSS request attempt 9... failure
[TSSR] Request URL set to http://gs.apple.com/TSS/controller?action=2
[TSSR] Sending TSS request attempt 10... failure
[TSSR] Request URL set to http://17.111.103.65/TSS/controller?action=2
[TSSR] Sending TSS request attempt 11... failure
[TSSR] Request URL set to http://17.111.103.15/TSS/controller?action=2
[TSSR] Sending TSS request attempt 12... failure
[TSSR] Request URL set to https://gs.apple.com/TSS/controller?action=2
[TSSR] Sending TSS request attempt 13... failure
[TSSR] Request URL set to https://17.111.103.65/TSS/controller?action=2
[TSSR] Sending TSS request attempt 14... failure
[TSSR] Request URL set to https://17.111.103.15/TSS/controller?action=2
[TSSR] Sending TSS request attempt 15... failure
Saved shsh blobs!
iOS 10.3.2 14F89 IS signed!

Build 14F89 for device iPhone7,2 IS being signed!

**Comment by tihmstar** [comment]
TL;DR
Everything is fine, it saved your blobs correctly. 



It says saved shsh blobs, looks good to me. 
When saving shsh files, tsschecker not only tries to save the APTicket you requested, but also additional APTickets like one for update install and one without a Nonce (so called noNonce-shsh files). 
There was a bug in iOS9 for 32bit devices, where the APNonce wasn't checked in case there is none in the APTicket, thus noNonce blobs allowed to re-restore/downgrade 32bit devices without jailbreak. 
I realized you can request similar tickets for 64bit devices and added that feature for fun. 
I didn't expect that to work in first place, but it seems that for some device you can get tickets without a nonce. So what tsschecker does, it saves noNonce tickets for these devices and shows that error for devices where it doesn't work. Right now there is no advantage of these blobs compared to normal ones, but you'll never know if there will be a bug in future.
